src/pages/Home/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './index.scss';
+
+const Home = (props) => {
+
+  return (
+    <main className="home">
+      <section className="start">
+        <div className="start__banner">
+
+        </div>
+        <div className="container start__categories">
+
+        </div>
+      </section>
+      <section className="container new-goods">
+        {/* <Goods
+          heading=""
+          link="#"
+          data={null}
+        /> */}
+      </section>
+      <section className="container popular-goods">
+        {/* <Goods
+          heading=""
+          link="#"
+          data={null}
+        /> */}
+      </section>
+      <section className="our-works">
+        {/* <Slider>
+
+        </Slider> */}
+      </section>
+      <section className="container why-are-we">
+
+      </section>
+      <section className="our-works">
+        {/* <Slider>
+          
+        </Slider> */}
+      </section>
+    </main>
+  )
+}
+
+export default Home;
```

Comment:
add empty line

Reply:
We need empty files, because they're gonna be filled soon. I'm just showing structure, how we gonna store logic. Because obviously there're multiple ways how to approach it

Reply:
and the empty line is added

Reply:
1 -  YAGNI principle says that you aren't gonna need it. Describe: not do anything that will be need in future, because in future you may not need it.
2 -  Structure of project should be described in architechure diagram and docs. It senselessly to create pr for creation of project structure.